components/DateTimePicker/DatePicker.tsx

```diff
@@ -0,0 +1,49 @@
+'use client';
+
+import { useState } from 'react';
+import { format } from 'date-fns';
+
+import { Calendar } from '@/components/ui/calendar';
+import InputField from '@/components/InputField/InputField';
+
+/**
+ * DatePicker 컴포넌트는 달력을 표시하고 선택한 날짜를 반환합니다.
+ * @param {string} width - optional, DatePicker 컴포넌트의 너비를 설정합니다. (기본 값은 w-full입니다.)
+ * @returns {JSX.Element} DatePicker 컴포넌트를 반환합니다.
+ */
+
+export default function DatePicker({ width = 'full' }: { width?: string }) {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
```

Comment:
w-full을 의도하신것같은데 하단에 코드리뷰를 추가 할 경우 디폴트값은 필요하지않을것 같습니다!

```suggestion
export default function DatePicker({ width }: { width?: string }) {
```